manifests/logging-stack/output-systemd-elasticsearch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: logging.kubesphere.io/v1alpha2
+kind: Output
+metadata:
+  name: systemd
+  namespace: kubesphere-logging-system
+  labels:
+    logging.kubesphere.io/enabled: "true"
+    logging.kubesphere.io/component: logging
+spec:
+  match: service.*
```

Comment:
Is it possible to use one single output for each output type by setting `match: kube.*|service.*` in the following file？It'll be convenient to modify one single output in UI

https://github.com/kubesphere/fluentbit-operator/blob/master/manifests/logging-stack/output-elasticsearch.yaml
https://github.com/kubesphere/fluentbit-operator/blob/master/manifests/logging-stack/output-forward.yaml
https://github.com/kubesphere/fluentbit-operator/blob/master/manifests/logging-stack/output-kafka.yaml